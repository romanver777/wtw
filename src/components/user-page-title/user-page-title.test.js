import React from "react";
import { shallow } from "enzyme";
import UserPageTitle from "./user-page-title";

describe("UserPageTitle component", () => {
  const props = {
    title: "Some text",
  };
  const Component = shallow(<UserPageTitle {...props} />);

  it("renders correctly", () => {
    expect(Component).toMatchSnapshot();
  });
  it("renders h1 once", () => {
    expect(Component.find("h1")).toHaveLength(1);
  });
  it("renders h1 with props.text", () => {
    expect(Component.find("h1.page-title.user-page__title").text()).toEqual(
      props.title
    );
  });
});
