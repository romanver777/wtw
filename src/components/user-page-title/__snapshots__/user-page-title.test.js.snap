// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPageTitle component renders correctly 1`] = `
<h1
  className="page-title user-page__title"
>
  Some text
</h1>
`;
